app/main.go

```diff
@@ -122,6 +117,11 @@ func main() {
 	}
 	logger.Info("Starting sidecar query server")
 
+	// If fails, it means that the node is not reachable
```

Comment:
Is this change intentional?

Reply:
Yes, this is so that we can get a log "Starting sidecar query server" without the dependency on the node